back/app/services/local_project_copy_service.rb

```diff
@@ -25,6 +26,18 @@ def copy(source_project)
     source_project.projects_topics.each { |projects_topic| projects_topic.dup.update!(project_id: copied_project.id) }
     source_project.areas_projects.each { |areas_project| areas_project.dup.update!(project_id: copied_project.id) }
 
+    LogActivityJob.perform_later(
```

Comment:
We usually log activities in `SideFx*Service`. You could add a new `after_copy` method to `SideFxProjectService`.

Reply:
Good point. i have moved it to a new `side_fx.after_copy` method.